vllm/config.py

```diff
@@ -1619,13 +1619,12 @@ def _verify_args(self) -> None:
         if self.use_ray:
             from vllm.executor import ray_utils
             ray_utils.assert_ray_available()
-        device_capability = current_platform.get_device_capability()
-        if (current_platform.is_rocm() and device_capability is not None
-                and device_capability < (9, 4)):
+
+        if current_platform.use_custom_allreduce():
```

Comment:
This condition should be inverted?
```suggestion
        if not current_platform.use_custom_allreduce():
```